composer.json

```diff
@@ -18,7 +18,7 @@
         "source": "https://github.com/cakephp/cakephp"
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.6.0",
```

Comment:
We might have to update the zip files we use on AppVeyor as well. I think @lorenzo knows how that is done.